ocrdmonitor/processstatus.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import annotations
 
-import subprocess
 from dataclasses import dataclass
 from datetime import timedelta
 from enum import Enum
 
-PS_CMD = "ps -g {} -o pid,state,%cpu,rss,cputime --no-headers"
+PS_CMD = "ps -s `cat /data/{}/ocrd.pid` -o pid,state,%cpu,rss,cputime --no-headers"
```

Comment:
Please separate the reading of the process ID from the `ps` command. 
By calling `cat` together with `ps` we're mixing two different concerns. It would be better to read the process ID from the file in Python and simply inject it into the format string.

Reply:
https://github.com/slub/ocrd_monitor/pull/24/commits/90987f90c73a3b0b9ab369aebfa468a8a6f3a315 – like this?